policy/policy.go

```diff
@@ -174,52 +181,71 @@ func (p *PUPolicy) AddTransmitterRules(t *TagSelector) {
 	p.transmitterRules.TagSelectors = append(p.transmitterRules.TagSelectors, *t.Clone())
 }
 
-// PolicyTags returns a copy of PolicyTag(s)
-func (p *PUPolicy) PolicyTags() *TagsMap {
+// Identity returns a copy of the Identity
+func (p *PUPolicy) Identity() *TagsMap {
+	p.puPolicyMutex.Lock()
+	defer p.puPolicyMutex.Unlock()
+
+	return p.identity.Clone()
+}
+
+// Annotations returns a copy of the annotations
+func (p *PUPolicy) Annotations() *TagsMap {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	return p.policyTags.Clone()
+	return p.annotations.Clone()
 }
 
-// SetPolicyTags sets up policy tags
-func (p *PUPolicy) SetPolicyTags(t *TagsMap) {
+// SetIdentity  sets up policy tags
+func (p *PUPolicy) SetIdentity(t *TagsMap) {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	p.policyTags = t.Clone()
+	p.identity = t.Clone()
 }
 
-// AddPolicyTag adds a policy tag
-func (p *PUPolicy) AddPolicyTag(k, v string) {
+// SetAnnotations  sets up annotations
+func (p *PUPolicy) SetAnnotations(t *TagsMap) {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	p.policyTags.Tags[k] = v
+	p.annotations = t.Clone()
+}
+
+// AddIdentityTag adds a policy tag
+func (p *PUPolicy) AddIdentityTag(k, v string) {
+	p.puPolicyMutex.Lock()
+	defer p.puPolicyMutex.Unlock()
+
+	p.identity.Tags[k] = v
 }
 
 // IPAddresses returns all the IP addresses for the processing unit
-func (p *PUPolicy) IPAddresses() *IPList {
+func (p *PUPolicy) IPAddresses() *IPMap {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	return p.policyIPs.Clone()
+	return p.ips.Clone()
 }
 
 // SetIPAddresses sets the IP addresses for the processing unit
-func (p *PUPolicy) SetIPAddresses(l *IPList) {
+func (p *PUPolicy) SetIPAddresses(l *IPMap) {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	p.policyIPs = l.Clone()
+	p.ips = l.Clone()
 }
 
 // DefaultIPAddress returns the default IP address for the processing unit
-func (p *PUPolicy) DefaultIPAddress() (string, bool) {
+func (p *PUPolicy) DefaultIPAddress() string {
 	p.puPolicyMutex.Lock()
 	defer p.puPolicyMutex.Unlock()
 
-	return p.policyIPs.IPAtIndex(0)
+	if ip, ok := p.ips.IPs[DefaultNamespace]; ok {
```

Comment:
Could 0.0.0.0/0 where we dont know an IP of a container affect all the traffic getting redirected here ?